Unix/scripts/config_keytab_update.sh

```diff
@@ -54,7 +54,9 @@ unconfigure()
 
     crontab -l > $tmpfile 2> /dev/null || true
 
-    egrep -v "$omikeytab" $tmpfile | crontab
+    tmpfile2=$(mktemp)
+    egrep -v "$omikeytab" $tmpfile >$tmpfile2 
```

Comment:
Change this to `grep`, you're not using regular expressions here.

It's really dumb, on the Sles-10 platform, that we get comments in the file. But I don't see a way around this.